docs/adr/0032-credential-store-canonical-home.md

```
@@ -0,0 +1,319 @@
+Storage's `credential` module imports the trait + DTOs from
+`nebula-credential` via sibling dep:
+
+```rust
+// crates/storage/src/credential/layer/encryption.rs
+use nebula_credential::{CredentialStore, StoredCredential, PutMode, StoreError};
+use nebula_credential::{CredentialId, CredentialRecord, encrypt_with_aad, decrypt_with_aad, ...};
+```
+
+### 3. One-directional dep graph
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)